app/models/cash_transaction.rb

```diff
@@ -1,9 +1,9 @@
 class CashTransaction < ApplicationRecord
   belongs_to :cash_transaction_category
 
-  scope :by_operation_and_category, lambda { |gross_income_summary, operation, category_name|
+  scope :by_operation_and_category, lambda { |operation, category_name|
     joins(:cash_transaction_category)
-      .where(cash_transaction_category: { name: category_name, operation:, gross_income_summary_id: gross_income_summary.id })
+      .where(cash_transaction_category: { name: category_name, operation: })
       .order(:date)
```

Comment:
This change makes me nervous (doubly-so when mixed in with a story). I'm worried that this breaks something that we don't have test coverage for (and might mix up cash transactions with someone elses)